scripts/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+## load variables from the env file
+source vars
```

Comment:
It would be nice if we could establish a convention around what to call this file that contains environment variables.  Having a file named ".env" in the same directory is the convention that docker-compose uses - maybe that's a good convention for us to use as well?  I'm not familiar with a convention using the name "vars" - is this something you've seen elsewhere @jmbabazi ?  @cioan - you also created a file for your script that performs similar and called it "env_vars".  Let's use this opportunity to do some research on a best practice convention that is used and then establish this consistently among these projects.